woocommerce/woocommerce-ios
# [Shipping Labels] Make loadPackages async

Closes: #14709


## Description


- Make `loadPackages()` method `async` for easier usage with `.refreshable` (pull to refresh) modifier

## Testing information


1. Install and set up the Woo Shipping extension on your store.
2. Build and run the app with the revampedShippingLabelCreation feature flag enabled.
3. Create an order with the processing status and at least one physical product.
4. In your WooCommerce product settings on the web, change the weight and dimension units and save your changes.
5. On the Orders tab in the app, open the order you created.
6. In the order details, select "Create Shipping Label."
7.  Tap "Select a Package."
8. Switch to Carrier tab and check that the packages are loaded
9. Pull to refresh and check that refreshing finishes and packages are loaded
10. Switch to Saved tab and check that the packages are loaded
11. Pull to refresh and check that refreshing finishes and packages are loaded

## Screenshots


---

## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
